packages/app/src/app/pages/Dashboard/Components/TemplatesRow/TemplatesRow.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { Icon, SkeletonText, Stack, Text } from '@codesandbox/components';
+import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { useOfficialTemplates } from 'app/components/CreateSandbox/useOfficialTemplates';
+import { TemplateCard } from 'app/components/CreateSandbox/TemplateCard';
+import { TemplateButton } from 'app/components/CreateSandbox/elements';
+import { GitHubIcon } from 'app/components/CreateSandbox/Icons';
+import { useActions, useAppState } from 'app/overmind';
+import { TemplateFragment } from 'app/graphql/types';
+import { GUTTER } from '../VariableGrid';
+import { TemplatesGrid } from './elements';
+
+export const TemplatesRow: React.FC = () => {
+  const officialTemplates = useOfficialTemplates();
+  const actions = useActions();
+  const { dashboard } = useAppState();
+
+  const collection = dashboard.allCollections.find(c => c.path === '/');
+
+  const handleOpenTemplate = (template: TemplateFragment) => {
+    const { sandbox } = template;
+    const url = sandboxUrl(sandbox);
+    window.open(url, '_blank');
+  };
+
+  const handleSelectTemplate = (template: TemplateFragment) => {
+    const { sandbox } = template;
+
+    actions.editor.forkExternalSandbox({
+      sandboxId: sandbox.id,
+      openInNewWindow: false,
+      body: {
+        collectionId: collection?.id,
+      },
+    });
+
+    actions.modals.newSandboxModal.close();
+  };
+
+  return (
+    <Stack
+      as="section"
+      css={css({
+        width: `calc(100% - ${2 * GUTTER}px)`,
+        marginX: 'auto',
+      })}
+      direction="vertical"
+      gap={4}
+    >
+      <Text
+        as="h2"
+        css={css({
+          margin: 0,
+        })}
+        weight="400"
+        size={4}
+      >
+        Pick up where you left off
+      </Text>
+      <TemplatesGrid>
+        {officialTemplates.state === 'loading' &&
+          new Array(5).fill(undefined).map((_, i) => (
+            <SkeletonText
+              // eslint-disable-next-line react/no-array-index-key
+              key={`templates-skeleton-${i}`}
+              css={css({
+                width: '100%',
+                height: '116px',
+              })}
```

Comment:
@tristandubbeld we probably should define a pattern for this. I prefer to always use the `css` function rather than having mixed usage in different places.

Reply:
@alexnm what do you prefer? We didn't really decide on how to use the `css` prop when talking about component styles and reuse.